downloads/swan-lake-release-notes/swan-lake-2201.3.0/RELEASE_NOTE.md

```diff
@@ -127,9 +127,9 @@ Previously, `-9223372036854775808` was allowed to be assigned to an integer in B
 int result = -9223372036854775808; // error: '9223372036854775808' is out of range for 'int'
 ```
 
-#### Restricted the typed binding of the query expression outer join to `var`
+#### Restriction on the typed binding pattern of an outer join clause to use `var`
 
-Previously, the typed binding for outer join could be any type descriptor, but now it's restricted to be `var` so that it will be inferred as a nillable type.
+Previously, the typed binding pattern in an outer join clause was allowed to be any type descriptor, but now it has been restricted to be `var` so that an optional type (`T?`) will be inferred as the type.
```

Comment:
```suggestion
Previously, the typed binding pattern in an outer join clause was allowed to be any type descriptor, but now it has been restricted to allow only `var`, so that an optional type (`T?`) will be inferred as the type.
```